desktop/e2e/tests/lib/driver-helper.js

```diff
@@ -291,7 +297,7 @@ exports.selectElementByText = async function ( driver, selector, text ) {
 			async ( e ) => ( await e.getText() ) === text
 		);
 	};
-	return await this.clickWhenClickable( driver, element );
+	return await this.clickWhenClickable( driver, element, `while looking for '${ text }'` );
```

Comment:
Nice.

Reply:
Yeah, this should help diagnosing future issues like this. It took me a while to figure out where that error was coming from.